deployments/go.mod

```diff
@@ -12,7 +12,7 @@ replace (
 require (
 	github.com/clarketm/json v1.17.1
 	github.com/kubearmor/KubeArmor/KubeArmor v0.0.0-20221130063629-a2dd0301b858
-	github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-20220908103453-7b92e248beb9
+	github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-00010101000000-000000000000
```

Comment:
Is this change intentional? (Though it doesn't matter because we have a replace directive for this anyway)